### PR TITLE
Hot fixes

### PR DESCRIPTION
**Pull request by bplantico**
What does this PR do?

This request validates title when an administrator creates a tutorial.

### CODE REVIEW

File: app/controllers/admin/tutorials_controller.rb [hunk 1]
```diff
@@ -4,6 +4,15 @@ def edit
   end
 
   def create
+    @tutorial = Tutorial.new(tutorial_params)
+
+    if @tutorial.save
+      flash[:success] = 'Successfully created tutorial.'
+      redirect_to tutorial_path(id: @tutorial.id)
```
Comment (alexander-mathieu) [inline app/controllers/admin/tutorials_controller.rb]: 👍

File: spec/features/admin/new_tutorial_spec.rb [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      expect(page).to have_content('How to Learn like a Learner')
+      expect(page).to have_content('There are no videos in this tutorial yet!')
+
+      visit root_path
+
+      expect(page).to have_content('Now with new Learnings!')
+      expect(page).to have_css("img[src='https://i.ytimg.com/vi/Drqj67ImtxI/hqdefault.jpg']")
+    end
+
+    it 'I fail to create a tutorial if I provide invalid information' do
```
Comment (alexander-mathieu) [inline spec/features/admin/new_tutorial_spec.rb:33]: Test looks good!

**Review by alexander-mathieu — APPROVED** [review]

File: app/models/tutorial.rb [hunk 3]
```diff
@@ -2,8 +2,8 @@ class Tutorial < ApplicationRecord
   has_many :videos, -> { order(position: :asc) }, dependent: :destroy
   acts_as_taggable_on :tags, :tag_list
   accepts_nested_attributes_for :videos
-
-  validates_format_of :thumbnail, with: %r{ (\Ahttps:\/\/i\.ytimg\.com\/vi\/([a-zA-Z0-9_-]*)\/hqdefault\.jpg\Z) }, on: :create
```
Comment (BrennanAyers) [inline app/models/tutorial.rb:6]: Hey we should keep the regex in here, as that's how I read the spec on #1 : 
`And I fill in 'thumbnail' with a valid YouTube thumbnail`

File: spec/features/admin/new_tutorial_spec.rb [hunk 4]
```diff
@@ -30,10 +30,9 @@
...
     end
 
-    it 'I fail to create a bad tutorial' do
+    it 'I fail to create a tutorial if I provide invalid information' do
       visit new_admin_tutorial_path
-
-      fill_in 'tutorial[title]', with: 'How to Learn like a Learner'
+      
       fill_in 'tutorial[description]', with: 'Now with new Learnings!'
       fill_in 'tutorial[thumbnail]', with: 'https://www.notavalid.com/youtube/thumbnmail.jpg'
```
Comment (BrennanAyers) [inline spec/features/admin/new_tutorial_spec.rb:37]: Like this part here

File: spec/models/tutorial_spec.rb [hunk 5]
```diff
@@ -1,6 +1,10 @@
 require 'rails_helper'
 
 RSpec.describe Tutorial, type: :model do
+  describe 'validations' do
+    it { should validate_presence_of :title }
+  end
```
Comment (BrennanAyers) [inline spec/models/tutorial_spec.rb:6]: We should already have a validations block down below?

**Review by BrennanAyers — CHANGES_REQUESTED** [review]

**Pull request merged**